docs/docs/guides/automate/sensors/run-status-sensors.md

```diff
@@ -4,6 +4,10 @@ sidebar_position: 500
 title: Creating sensors that react to run statuses
 ---
 
+import ScaffoldSensor from '@site/docs/partials/\_ScaffoldSensor.md';
```

Comment:
This might not make sense here, since this page is about the `@run_status_sensor` decorator, which doesn't seem scaffold-able via the dg CLI